main.tf

```diff
@@ -0,0 +1,45 @@
+# Installs and configures cert-manager
```

Comment:
In my opinion superflous comment - i mean the module is named certmanager and the README says clearly what it is. We don't need that comment in my opinion...